src/reconstruction/refinement/mergeModelFieldPositions.m

```diff
@@ -0,0 +1,171 @@
+function modelNew = mergeModelFieldPositions(model,type,positions)
```

Comment:
Do you have any test for `mergeModelFieldPositions`?

Reply:
Thanks for the request @laurentheirendt, otherwise I would have missed some bugs.

Reply:
Great! 👍